src/framework/entity.js

```diff
@@ -59,7 +59,7 @@ Object.assign(pc, function () {
             }
         }
 
-        this.setGuid(pc.guid.create()); // Globally Unique Identifier
+        this._guid = null;
```

Comment:
I feel like this impacts findByGuid() which still uses this._guid? I'm guessing this works because guid is created before anyone could successfully call findByGuid?